recipes/libpq/config.yml

```diff
@@ -1,14 +1,24 @@
 versions:
+  "13.2":
+    folder: all
   "13.1":
     folder: all
   "13.0":
     folder: all
+  "12.6":
+    folder: all
+  "12.5":
+    folder: all
   "12.4":
     folder: all
   "12.3":
     folder: all
   "12.2":
     folder: all
+  "11.11":
+    folder: all
+  "11.10":
+    folder: all
```

Comment:
Is it not sufficient to add 11.11? Same question for 12.5 and 12.6?

Reply:
Who knows, there might regressions, for example.

Reply:
If that should be the case you can still add them later if actually needed